contracts/ColonyNetwork.sol

```diff
@@ -82,8 +82,9 @@ contract ColonyNetwork is ColonyNetworkStorage {
     return colonyVersionResolver[_version];
   }
 
-  function getSkill(uint256 _skillId) public view returns (uint256, uint256) {
-    return (skills[_skillId].nParents, skills[_skillId].nChildren);
+  function getSkill(uint256 _skillId) public view returns (uint256, uint256, bool) {
+    Skill storage skill = skills[_skillId];
```

Comment:
@elenadimitrova Looks doable. Only thing is we would need to extract the `Skill` type definition from `ColonyNetworkStorage` and place it in a library so `iColonyNetwork.sol` can access the `Skill` type for function return declaration. Thoughts?

Reply:
I think this might warrant a bigger refactoring piece with some upgrades testing. Might log it separately so we don't have to invent the design here. I like the library idea though. Will be the likely winner as to how we implement this but will need to consider upgrade options, e.g. what happens when a new struct member is added, can `EtherRouter` handle that without reregistering in the `Resolver` and upgrading the contract implementation, e.g. the `ColonyNetwork` in the above case.

Reply:
Ok sounds good to me. Moving this back to `ready-for-review` as I have addressed the other comment regarding moving the tests.

Reply:
Now logged separately in https://github.com/JoinColony/colonyNetwork/issues/283